openstack_commands/node_manage.sh

```diff
@@ -26,6 +38,8 @@ if [[ $1 == "prepare" ]]; then
         openstack baremetal port create --node $uuid ${!mac}  --local-link-connection switch_info=switch1 --local-link-connection switch_id=4c:d9:8f:dc:e0:c9 --local-link-connection port_id=Te1/$port --physical-network datacentre
         openstack baremetal node set  --property capabilities=iscsi_boot:True oct4-$n
         openstack baremetal node set --instance-info storage_interface=cinder oct4-$n
+        openstack baremetal node set --driver-info ipmi_terminal_port=$free_port oct4-$n
```

Comment:
Do we also want to enable console here? "openstack baremetal node console enable"

Reply:
Nope - we want to leave it to the lessee to explicitly enable/disable the console. It does provide an access point to the node that not everyone will want. In fact, it's safest to disable the console as soon as the lessee is done with it.

Reply:
Gotcha!